### PR TITLE
Label tests with openshift crafted API groups (round 1)

### PR DESCRIPTION
**Pull request by ingvagabund**
Procedure for detecting API groups in use:
- search for all Get, Update, Patch, Delete and List client invocations
  (for both direct and indirect)
- go through every e2e test code and identify all possible invocations of an api group (e.g. oc command)

jira: https://issues.redhat.com/browse/WRKLDS-488

### CODE REVIEW

File: test/extended/authorization/scc.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ var _ = g.Describe("[sig-auth][Feature:SCC][Early]", func() {
 
 	oc := exutil.NewCLIWithoutNamespace("working-scc-during-install")
 
-	g.It("should not have pod creation failures during install", func() {
+	g.It("should not have pod creation failures during install [apigroup:config.openshift.io]", func() {
```
Comment (deads2k) [inline test/extended/authorization/scc.go:24]: What requires this api group in this test?

File: test/extended/apiserver/root_403.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ var _ = g.Describe("[sig-api-machinery][Feature:APIServer]", func() {
 
 	oc := exutil.NewCLI("apiserver")
 
-	g.It("anonymous browsers should get a 403 from /", func() {
+	g.It("anonymous browsers should get a 403 from / [apigroup:config.openshift.io]", func() {
```
Comment (deads2k) [inline test/extended/apiserver/root_403.go:28]: this test should be rewritten to avoid the dep

File: test/extended/authorization/scc.go (hunk 1 above)
Comment (ingvagabund) [reply test/extended/authorization/scc.go:24]: `util.AllClusterVersionsAreGTE` and https://github.com/ingvagabund/origin/blob/7826178007475886fb8878e1e60e91f14eb476b2/test/extended/util/framework.go#L1986:
```
func AllClusterVersionsAreGTE(version semver.Version, config *rest.Config) (bool, error) {
	c, err := configv1client.NewForConfig(config)
	if err != nil {
		return false, err
	}
	cv, err := c.ConfigV1().ClusterVersions().Get(context.TODO(), "version", metav1.GetOptions{})
	if err != nil {
		return false, err
	}
```

Comment (deads2k) [reply test/extended/authorization/scc.go:24]: Fun.  Lucky for us, we can remove that line now.

Comment (ingvagabund) [reply test/extended/authorization/scc.go]: Removed

File: test/extended/apiserver/root_403.go (hunk 2 above)
Comment (ingvagabund) [reply test/extended/apiserver/root_403.go]: Removed

**Comment by ingvagabund** [comment]
Just re-generating go.mod

**Pull request merged**